src/adapter.c

```diff
@@ -655,14 +655,25 @@ AdapterCreate(
         //
```

Comment:
The commit message still says:
Add NDIS_MINIPORT_ATTRIBUTES_NO_PAUSE_ON_SUSPEND
:)